pkg/config/config_template.yaml

```
@@ -413,6 +413,125 @@ api_key:
 #
 # secret_backend_timeout: 5
 
+## @param snmp_listener - custom object - optional
+## Creates and schedules an snmp listener to automatically discover your SNMP devices.
+## Discover devices can then be monitored with the snmp integration by using
+## the provided auto_conf.yaml file.
+#
+# snmp_listener:
+
+  ## @param workers - integer - optional - default: 2
+  ## Number of goroutines used to discover SNMP devices. Increasing this value
```

Is "goroutine" an implementation detail? Maybe "concurrent tasks":

```suggestion
  ## Number of concurrent tasks used to discover SNMP devices. Increasing this value
```